packages/plugin/src/theia.d.ts

```diff
@@ -4355,512 +4520,380 @@ declare module '@theia/plugin' {
         readonly index: number;
     }
 
-    /**
-     * Enumeration of file types. The types `File` and `Directory` can also be
-     * a symbolic links, in that use `FileType.File | FileType.SymbolicLink` and
-     * `FileType.Directory | FileType.SymbolicLink`.
-     */
+	/**
+	 * Enumeration of file types. The types `File` and `Directory` can also be
+	 * a symbolic links, in that case use `FileType.File | FileType.SymbolicLink` and
```

Comment:
"...can also be a symbolic _link_..." while we're at it.

Reply:
fixed